core/src/Borrower.sol

```diff
@@ -277,6 +293,7 @@ contract Borrower is IUniswapV3MintCallback {
         if (amount1 > 0) LENDER1.borrow(amount1, recipient);
     }
 
+    // TODO: change/reword this
```

Comment:
Instead of putting a TODO here, what if you just made a GitHub issue?

Reply:
Don't worry, I'll do another round of TODO cleanups soon :)